BSPConversionCmd/Program.cs

```diff
@@ -22,38 +22,77 @@ class Options
 			[Option("prefix", Required = false, HelpText = "Prefix for the converted BSP's file name.")]
 			public string Prefix { get; set; }
 
-			[Value(0, MetaName = "input file", HelpText = "Input Quake 3 BSP/PK3 to be converted.", Required = true)]
-			public string InputFile { get; set; }
+			[Option('i', "input", Required = false, HelpText = "Input file(s) and its path to be processed. (i.e. -input \"C:\\path\\to\\file.pk3)\" ", Separator = ',')]
+			public IEnumerable<string> InputFile { get; set; }
 
-			[Value(1, MetaName = "output directory", HelpText = "Output game directory for converted BSP/materials.", Required = true)]
+			[Option('o', "output", Required = false, HelpText = "Output game directory for converted BSP/materials. (i.e. -output \"C:\\path\\to\\output\\folder)\" ")]
 			public string OutputDirectory { get; set; }
 		}
 
 		static void Main(string[] args)
 		{
-			//args = new string[3];
-			//args[0] = @"c:\users\tyler\documents\tools\source engine\bspconvert\kopo_fe.pk3";
-			//args[1] = @"c:\users\tyler\documents\tools\source engine\bspconvert\output";
-			//args[2] = "--newbsp";
+            //args = new string[3];
+            //args[0] = @"c:\users\tyler\documents\tools\source engine\bspconvert\kopo_fe.pk3";
+            //args[1] = @"c:\users\tyler\documents\tools\source engine\bspconvert\output";
+            //args[2] = "--newbsp";
+
+			var strFullInput = "";
```

Comment:
You shouldn't have to do any argument parsing if you use `Value` instead of `Option` for the input file attribute.